discretize/utils/meshutils.py

```diff
@@ -12,6 +14,22 @@
     num_types = [int, float]
 
 
+def load_mesh(filename):
+    """
+    Open a json file and load the mesh into the target class
+
+    As long as there are no namespace conflicts, the target __class__
```

Comment:
If you want, you can narrow down your namespace a little by overriding the `_REGISTRY` on your `BaseMesh` class (ie just add `_REGISTRY = {}` to the class). Then you could say `data = BaseMesh.deserialize(...)` and it will only look at classes that inherit from `BaseMesh` (and therefore are on the BaseMesh registry, rather than the HasProperties registry).

Reply:
Ok, cool, thanks!